src/ansys/tools/meilisearch/scrapper.py

```
@@ -75,11 +82,25 @@ def _scrape_url_command(self, temp_config_file):
         -------
         str
             Output of scraping the URL for the web page.
+
+        Raises
+        ------
+        SubprocessExecutionError
+            If any error occurs during the subprocess execution.
         """
-        result = subprocess.run(
-            ["python", "-m", "scraper", temp_config_file], stdout=subprocess.PIPE
-        )
-        output = result.stdout.decode("utf-8")
+        try:
+            result = subprocess.run(
+                ["python", "-m", "scraper", temp_config_file],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+            )
```

FYI... this is a problem

In case you are using a virtual environment, the moment you execute a shell as subprocess, the environment is not activated. I know there is no easy way but... could we posibly call the scraper module from Python?  Instead of delegating it to the CLI entrypoint... That way, if it crashes, we can also forward the error and avoid "silent errors"...

https://github.com/ansys/pymeilisearch-scraper/blob/main/scraper/__main__.py